bedrock/settings/base.py

```diff
@@ -258,14 +258,39 @@ def data_path(*args):
 FLUENT_L10N_TEAM_REPO_PATH = DATA_PATH / "l10n-team"
 # 10 seconds during dev and 10 min in prod
 FLUENT_CACHE_TIMEOUT = config("FLUENT_CACHE_TIMEOUT", default="10" if DEBUG else "600", parser=int)
-# Order matters. first string found wins.
+# order matters. first sting found wins.
 FLUENT_PATHS = [
     # local FTL files
     FLUENT_LOCAL_PATH,
     # remote FTL files from l10n team
     FLUENT_REPO_PATH,
 ]
 
+# These are defined up front, because we need them for more than just Pocket mode, but
+# note that they are also swapped in as the Fluent defaults in settings/__init__.py
+POCKET_FLUENT_REPO = config(
+    "POCKET_FLUENT_REPO",
+    default="mozilla-l10n/pocket-www-l10n",
+)
+POCKET_FLUENT_REPO_URL = f"https://github.com/{POCKET_FLUENT_REPO}"
+POCKET_FLUENT_REPO_PATH = DATA_PATH / "pocket-www-l10n"
+POCKET_FLUENT_REPO_BRANCH = config("POCKET_FLUENT_REPO_BRANCH", default="main")
+
+# This config is used to ensure that l10n_update.py can pull from both, separate,
+# L10N repos for Mozorg and for Pocket and update the appropriate dirs
+FLUENT_L10N_UPDATE_PARAMS = {
```

Comment:
With this being so specific to `l10n_update.py` I think it makes more sense to be defined as a constant in that file. Just a suggestion though.